govcd/lbserverpool_test.go

```diff
@@ -90,6 +90,10 @@ func (vcd *TestVCD) Test_LBServerPool(check *C) {
 	// We created server pool successfully therefore let's add it to cleanup list
 	AddToCleanupList(TestLbServerPool, "lbServerPool", parentEntity, check.TestName())
 
+	// Try to delete used service monitor and expect it to fail with nice error
+	err = edge.DeleteLbServiceMonitor(lbMon)
+	check.Assert(err, ErrorMatches, ".*Fail to delete objectId .* for it is used by .*")
```

Comment:
Replace `objectId .*` with `objectId .*\S+.*` to make sure that there is some ID returned.

Reply:
Thanks. Improved and tested at least on 9.5 and 9.7